cmd/kind/load/docker-image/docker-image.go

```diff
@@ -100,43 +95,12 @@ func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	// Save the image into a tar
-	dir, err := fs.TempDir("", "image-tar")
-	if err != nil {
-		return errors.Wrap(err, "failed to create tempdir")
-	}
-	defer os.RemoveAll(dir)
-	imageTarPath := filepath.Join(dir, "image.tar")
-
-	err = docker.Save(args[0], imageTarPath)
-	if err != nil {
-		return err
-	}
-
-	// Load the image into every node
+	// Save the image and load it into every selected node.
 	for _, node := range selectedNodes {
-		if err := loadImage(imageTarPath, &node); err != nil {
-			return err
+		if err := node.LoadContainerImage(args[0]); err != nil {
```

Comment:
this is a pretty major performance regression because we now ave the image to a tarball once per node

Reply:
@BenTheElder thank you for catching that!
I think I fixed it